website/content/docs/enterprise/fips.mdx

```diff
@@ -0,0 +1,127 @@
+---
+
+# FIPS 140-2
+
+<Note> This feature requires Consul Enterprise. </Note>
+
+Builds of Consul Enterprise marked with a `fips1402` feature name include built-in support for FIPS 140-2 compliance.
+
+To use this feature, you must have an [active or trial license for Consul Enterprise](/consul/docs/enterprise/license/overview). To start a trial, contact HashiCorp sales.
+
+## Using FIPS 140-2 Consul Enterprise
+
+FIPS 140-2 versions of Consul Enterprise behave like non-FIPS versions of Consul. No restrictions are placed on algorithms and it is up to the operator to ensure Consul remains in a FIPS-compliant mode of operation. Maintaining FIPS-compliant operation requires configuring some settings such as [ensuring TLS is enabled to require encrypted communication](consul/tutorials/security/tls-encryption-secure). Consul products surface some helpful warnings where settings are insecure.
+
+Because Consul Enterprise’s default settings may transmit sensitive control plane information without encryption, it is especially important to ensure that all agents have gossip encryption and mTLS enabled when running Consul with FIPS-compliant settings. In addition, be aware that TLSv1.3 does not work with FIPS 140-2, as HKDF is not a certified primitive.
+HashiCorp is limited to providing general guidance regarding using Consul Enterprise in a FIPS-compliant manner. We are not a NIST-certified testing laboratory and thus organizations may need to consult an approved auditor for further information.
+
+The FIPS 140-2 variant of Consul uses separate binaries that are available available from the following sources:
+
+- From the [Hashicorp Releases page](https://releases.hashicorp.com/consul), releases ending with the `+ent.fips1402` suffix.
+- From the [Docker Hub `hashicorp/consul-enterprise-fips`](https://hub.docker.com/r/hashicorp/consul-enterprise-fips) container repository.
+- From the [AWS ECR `hashicorp/consul-enterprise-fips`](https://gallery.ecr.aws/hashicorp/consul-enterprise-fips) container repository.
+- From the [Red Hat Access `hashicorp/consul-enterprise-fips`](https://catalog.redhat.com/software/containers/hashicorp/consul-enterprise-fips/628d50e37ff70c66a88517ea) container repository.
+
+The above naming conventions, which append `.fips1402` to binary names and tags, and `-fips` to registry names, also apply to `consul-k8s`, `consul-dataplane`, and `consul-ecs`, which are packaged separately from Consul Enterprise.
+
+### Usage restrictions
+
+When using Consul Enterprise with FIPS 140-2, be aware of the following operation restrictions:
+
+#### Migration restrictions
+
+Hashicorp does not support in-place migrations from non-FIPS versions of Consul to FIPS versions of Consul, regardless of version. A fresh cluster installation is required to support FIPPS 140-2. It is not possible to upgrade directly.
```

Comment:
```suggestion
We do not support in-place migrations from non-FIPS builds of Consul to FIPS builds of Consul, regardless of version. A fresh cluster installation is required to support FIPS 140-2. You cannot upgrade directly to a FIPS-compliant build.
```
We should disambiguate between "build" and "version". Two builds of Consul (enterprise, OSS, FIPS) can be on the same version of Consul.